rocsp/config/rocsp_config.go

```diff
@@ -138,3 +146,80 @@ func MakeReadClient(c *RedisConfig, clk clock.Clock) (*rocsp.Client, error) {
 	})
 	return rocsp.NewClient(rdb, timeout, clk), nil
 }
+
+// A ShortIDIssuer combines an issuance.Certificate with some fields necessary
+// to process OCSP responses: the subject name and the shortID.
+type ShortIDIssuer struct {
+	*issuance.Certificate
+	subject pkix.RDNSequence
+	shortID byte
+}
+
+// LoadIssuers take a map where the keys are filenames and the values are the
+// corresponding short issuer ID. It loads issuer certificates from the given
+// files and produces a []ShortIDIssuer.
+func LoadIssuers(input map[string]int) ([]ShortIDIssuer, error) {
+	var issuers []ShortIDIssuer
+	for issuerFile, shortID := range input {
+		if shortID > 255 || shortID < 0 {
+			return nil, fmt.Errorf("invalid shortID %d (must be byte)", shortID)
+		}
+		cert, err := issuance.LoadCertificate(issuerFile)
+		if err != nil {
+			return nil, fmt.Errorf("reading issuer: %w", err)
+		}
+		var subject pkix.RDNSequence
+		_, err = asn1.Unmarshal(cert.Certificate.RawSubject, &subject)
+		if err != nil {
+			return nil, fmt.Errorf("parsing issuer.RawSubject: %w", err)
+		}
+		var shortID byte = byte(shortID)
+		for _, issuer := range issuers {
+			if issuer.shortID == shortID {
+				return nil, fmt.Errorf("duplicate shortID in config file: %d (for %q and %q)", shortID, issuer.subject, subject)
```

Comment:
This breaks our convention around only inlined errors coming after `:` at the end of a line. I also found it a little hard to read; this message seems to imply that `shortID` is the name of a config file.

Here's a slightly modified version that, I believe, resolves both of these nits.

```suggestion
				return nil, fmt.Errorf("duplicate shortID '%d' (for %q and %q) in config file", shortID, issuer.subject, subject)
```